src/GRA.Domain.Service/BadgeService.cs

```diff
@@ -68,7 +69,8 @@ public async Task<Badge> AddBadgeAsync(Badge badge, byte[] imageFile)
         public async Task<Badge> UpdateBadgeAsync(Badge badge)
         {
             badge.SiteId = GetCurrentSiteId();
-            return await _badgeRepository.AddSaveAsync(GetClaimId(ClaimType.UserId), badge);
+            badge.AltText = badge.AltText.Trim();
```

Comment:
badge.AltText should use null coalescing before Trim() to prevent any potential null exception.

Reply:
Fixed in commit 2a6e7e46aaeb865d223b5e52d3e55e5a766d89cd